code/datums/dna.dm

```diff
@@ -8,6 +8,7 @@
 	var/list/features = list("FFF") //first value is mutant color
 	var/real_name //Stores the real name of the person who originally got this dna datum. Used primarely for changelings,
 	var/list/mutations = list()   //All mutations are from now on here
+	var/original_wings = "Plain" //Used for the restoration of moth wings, also a failsafe
```

Comment:
comment should describe what the field is as well (the wings selected in their character)